recipes/cms-fireworks/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "fwlite" %}
+{% set version = "10.5.0" %}
+{% set hash = "2f5c169b6ad54330f8c8ffee8ac287c719b579c2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/gartung/fwlite.git
```

Comment:
Ideally this should be a tagged tarball with a SHA256 (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarballs#build-from-tarballs-not-repos)) and patches applied on top if required.